app/src/migrations/20200610000000-8-ipc-operationtype-col.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.addColumn(
+          'ipc_plan',
+          'operationType',
+          {
+            allowNull: true,
+            comment: 'Forestry Operation Type for this plan',
+            type: Sequelize.STRING(30),
+            unique: false,
+            defaultValue: 'SILVICULTURE'
```

Comment:
Not sure if already discussed, but Matt said that previous entries are already being made for non-silv types, so not sure if defaulting to silv or NULL is what we want... hopefully could get an answer quick from business if we need one

Reply:
oh. i'll check with Matt. null sounds better. that will affect the output